app/views/phases/_edit_phase.html.erb

```diff
@@ -1,22 +1,28 @@
 <%= form_for :phase, { url: admin_update_phase_path(phase.id), html: { method: :put }} do |f| %>
-  <div class="form-group col-xs-8">
-    <%= f.label(:title, _('Title') ,class: 'control-label') %>
+  <div class="form-group col-md-10">
+    <%= f.label(:title, _('Title') ,class: "control-label") %>
     <%= f.text_field(:title, class: "form-control", 'aria-required': false, 'data-toggle': 'tooltip', title: _('Enter a title for the phase e.g. intial DMP, full DMP... This is what users will see in the tabs when completing a plan. If you only have one phase, call it something generic e.g. Glasgow DMP')) %> 
   </div>
 
-  <div class="form-group col-xs-8">
-    <%= f.label(:number, _('Order of display'), class: 'control-label') %>
-    <%= f.number_field(:number, in: 1..5, class: "form-control", 'aria-required': false, 'data-toggle': 'tooltip', title: _('This allows you to order the phases of your template.')) %>
+  <div class="form-group">
+    <div class="col-md-10">
+      <%= f.label(:number, _('Order of display'), class: "control-label") %>
+    </div>
+    <div class="col-md-4">
+      <%= f.number_field(:number, in: 1..5, class: "form-control", 'aria-required': false, 'data-toggle': 'tooltip', title: _('This allows you to order the phases of your template.')) %>
+    </div>
   </div>
 
-  <div class="form-group col-xs-8">
-    <%= f.label(:description, _('Description'), class: 'control-label') %>
+  <div class="form-group col-md-10">
+    <%= f.label(:description, _('Description'), class: "control-label") %>
     <div data-toggle="tooltip" title="<%= _("Enter a basic description. This will be presented to users on the 'Admin Plan' tab, above the summary of the sections and questions which they will be asked to answer.") %>">
-      <%= text_area_tag('phase-desc', phase.description, class: 'phase form-control') %>
+    <%= text_area_tag('phase-desc', phase.description, class: 'phase tinymce form-control') %>

```

Comment:
phase class is used for loading tinymce instances already. Not need for adding the class tinymce (e.g. lib/assets/javascripts/views/phases/edit.js L8